README.md

```diff
@@ -1,15 +1,17 @@
-# BlueRobotics TSYS01 Temperature Sensor Library [![Build Status](https://travis-ci.org/bluerobotics/BlueRobotics_TSYS01_Library.svg?branch=master)](https://travis-ci.org/bluerobotics/BlueRobotics_TSYS01_Library)
+# BlueRobotics TSYS01 Temperature Sensor Library
 
 Arduino library for the TSYS01 temperature sensor. The TSYS01 is a high-accuracy, fast-response subsea temperature sensor from Measurement Specialties. 
 
 # Documentation
 
 Please see the examples for normal operation. Below are the available functions used in the library.
 
-``` cpp
+```ino
```

Comment:
Is there a reason to use ino ? This doesn't sound standard.

Reply:
It's the code for Arduino code snippets, and this is an Arduino library - just seemed to make sense to me?

Reply:
It's not important to me, so I can remove it if you don't think it makes sense for what it represents.

Reply:
Is there an advantage to do so ? I'm just afraid that others markdows renders that work with GitHub fail with this extension. If cpp works, and this is a cpp code, and Arduino is still programmed in c++ as well, and there is no further advantage, I don't see a reason for this change.

Reply:
> Is there an advantage to do so ?

There are some compilation differences for `.ino` vs `.cpp` files, but since this is just a snippet then no, I don't believe there are any differences in the highlighting.

I've changed it back now.